src/items/repositories/item.repository.ts

```diff
@@ -68,7 +68,7 @@ export class ItemRepository extends Repository<Item> {
 
   async getNewest(qty: number): Promise<Item[]> {
     return await this.getItemQueryBuilder()
-      .where('item.status=:status', { status: ItemStatus.Listed })
+      .where('item.status != :status', { status: ItemStatus.Hidden })
```

Comment:
what about draft items ? Is that even possible ?

Reply:
Good point. lemme fix it